pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1334,7 +1334,6 @@ func handleCreate(ctx *domainContext, key string, config *types.DomainConfig) {
 }
 
 // doAssignAdaptersToDomain assigns IO adapters to the newly created domain.
-// Note that the adapters are already reserved for the domain using reserveAdapters (UsedByUUID is set).
```

Comment:
Did you change reserveAdapters to not sed UsedByUUID?

Reply:
Looking at this some more you still set that, but the hyper.pcireserve is deferred. So it makes more sense to ammend the comment than to delete it.

Reply:
I changed the comment to:
```
// doAssignAdaptersToDomain assigns IO adapters to the newly created domain.                                                                                      
// The adapters are reserved here for the domain                                                                                                                  
// UsedByUUID is already set in reserveAdapters            
```

Reply:
Thanks